### DISCUSSION TITLE
Sort documents on the left sidebar

### DISCUSSION THREAD

**Discussion by mr-bantsevich** [outline/outline#1370] [General] [answered]
I use menu on left sidebar for navigate between documents.
And I want to sort documents inside this sidebar. Can you improve this feature? Thanks for great products!

![Снимок экрана 2020-07-20 в 18 00 32](https://user-images.githubusercontent.com/12588034/87952816-eac52700-cab2-11ea-8a16-4cf6a3fff285.png)

**Comment by tommoor** [comment, marked as answer]
Hey, the feature is under development here:
https://github.com/outline/outline/pull/1283
